src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/MetadataFields.scala

```diff
@@ -0,0 +1,42 @@
+package uk.gov.nationalarchives.tdr.api.graphql.fields
```

Comment:
I would be tempted to rename this: `CustomMetadataFields`, just to make it clearer what it is. The word `metadata` is used a lot in the code, and its going to be important to qualify what sort of metadata it is, to avoid confusion.